modern/src/resources/l10n/en.json

```diff
@@ -177,6 +177,8 @@
     "loginReset": "Reset Password",
     "loginRegister": "Register",
     "loginLogin": "Login",
+    "loginLoginOIDC": "Login with SSO",
```

Comment:
No all caps please. Also maybe name this something like "Login with OpenID"? I think it would be more clear for the end user.